released/packages/coq-waterproof/coq-waterproof.1.2.4/opam

```diff
@@ -0,0 +1,42 @@
+opam-version: "2.0"
+synopsis: "Waterproof library"
+description: """
+  The Waterproof library provides tactics, notations, and mathematical theories geared towards use in Mathematics educational environments. It aims to provide syntax such that proof scripts mimic handwritten mathematical proofs.
+"""
```

Comment:
```suggestion
description: """
The Waterproof library provides tactics, notations, and mathematical theories geared towards use
in Mathematics educational environments. It aims to provide syntax such that proof scripts mimic
handwritten mathematical proofs.
"""
```
How about some linebreaking here for better presentation? Just a suggestion.

Reply:
Thanks for the suggestion